FitNesseRoot/FitNesse/UserGuide/WritingAcceptanceTests/SliM/SlimProtocol/content.txt

```diff
@@ -146,6 +146,12 @@ If a fixture throws an exception with a ''class'' name that contains "!style_cod
 
 To abort the test suite (stop execution directly), throw an exception with a class name that contains "!style_code(!-StopSuite-!)". The syntax is identical to !-StopTest-!.
 
+!4 Ignoring Test
+If a fixture throws an exception with a ''class'' name that contains "!style_code(!-IgnoreScriptTest-!)", then Slim should ignore and skip the instructions in the rest of that script table.   The response for this type of exception should be "!style_code(__EXCEPTION__:IGNORE_SCRIPT_TEST:)" which may have an optional suffix of: "!style_code(message:<<''reason''>>)".
+
+!4 Ignoring All Tests
+If a fixture throws an exception with a ''class'' name that contains "!style_code(!-IgnoreAllTests-!)", then Slim should ignore and skip all the instructions in the rest of the current batch.   The response for this type of exception should be "!style_code(__EXCEPTION__:IGNORE_ALL_TESTS:)" which may have an optional suffix of: "!style_code(message:<<''reason''>>)".
```

Comment:
What do you mean with 'current batch', the current test page, or suite, or ...?

Reply:
@fhoeben  I used that term as it was used previously for the StopTest exception handling functionality. But I've reworded this line now in the last commit.